### DISCUSSION TITLE
How to use LQR to control TMD

### DISCUSSION THREAD

**Discussion by icnosvbovwb** [OpenFAST/openfast#1887] [Q&A]
Dear @jjonkman 
Can you tell me how to actively control TMD with LQR? The TMD is place on the platform to reduce platform pitch. But I am not familiar with LQR, how do I use it?

**Comment by jjonkman** [comment]
It looks like a similar question was asked and answered in the ROSCO repository: https://github.com/NREL/ROSCO/discussions/285.
